static/app/views/issueDetails/streamline/sidebar/seerNotices.tsx

```diff
@@ -55,18 +55,12 @@ function GithubIntegrationSetupCard() {
   );
 }
 
-export function SeerNotices({
-  autofixRepositories,
-  hasGithubIntegration,
-}: SeerNoticesProps) {
+export function SeerNotices({groupId, hasGithubIntegration}: SeerNoticesProps) {
```

Comment:
should we add a notice for the case where you have no repos in settings and no repos from code mappings?

Reply:
Good call out, I'll do that in a follow up PR where I do the flow for onboarding the repo settings